includes/sanitizers/class-amp-video-sanitizer.php

```diff
@@ -134,6 +134,11 @@ public function sanitize() {
 			$new_attributes = $this->filter_attachment_layout_attributes( $node, $new_attributes, $layout );
 			if ( empty( $new_attributes['layout'] ) && ! empty( $new_attributes['width'] ) && ! empty( $new_attributes['height'] ) ) {
 				$new_attributes['layout'] = 'responsive';
+
+				// The wp-block-cover__video-background class applies the 'object-fit: cover' CSS property, so we need to set the layout to 'fill'.
+				if ( $node->hasAttribute( 'class' ) && in_array( 'wp-block-cover__video-background', explode( ' ', $node->getAttribute( 'class' ) ), true ) ) {
+					$new_attributes['layout'] = 'fill';
+				}
```

Comment:
Perhaps good to cherry-pick into another PR?

Reply:
I'm having trouble replicating this issue with latest `develop`. Will revisit later, it might be solved through a different PR already...